### ISSUE TITLE
The use of trained checkpoint 

### ISSUE THREAD

**Issue by Shengyu-Feng** [google-research/football#83]
For the trained checkpoint, is there any way to use them in the ppo agent in _baselines_ library? I've tried to set them in the  "load_path" in the example code, but it doesn't seem to work.

Btw: I've use the example code to train ppo agent in 11_vs_11_easy_stochastic, but I didn't get the expected result in the paper, do I need to additionally set some parameters?

**Comment by lucasb-eyer** [comment]
Hi, I was able to reproduce it exactly on cloud, I plan to put the configs for reproduction in this repo in a few days (currently busy with ICLR sorry)

**Comment by kkurach** [comment]
Yes, the default example should work without any changes... @Shengyu-Feng , which reward do you use? How many steps do you train?  And, what's the results that you get?

**Comment by Shengyu-Feng** [comment]
Oh, my bad. I forgot to reset the hyperparamters, it works right now, I'll close it.
